lib/generate.js

```diff
@@ -15,7 +15,8 @@ const generators = {
   maxWidths: require('./max-widths'),
   heights: require('./heights'),
   typography: require('./typography'),
-  opacity: require('./opacity')
```

Comment:
I wasn't sure if there was a reason for the ordering of these properties without digging into the source further — can these be alphabetized, or are these `require`s order-dependent?

Reply:
We can definitely alphabetize here. Only the end snapshot is order dependent, so we can simply update that and all is good!